Public/html/home.html

```diff
@@ -1,53 +1,64 @@
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home | Investment App</title>
+  <link rel="stylesheet" href="../css/home.css">
 </head>
-<body>
-    <!-- Navigation Bar -->
-    <nav class="navbar">
-        <a href="#" class="logo">Investment App</a>
-        <div class="nav-links">
-            <a href="home.html">Home</a>
-            <a href="portfolio.html">My Portfolio</a>
-            <a href="notifications.html">Notifications</a>
-            <a href="trending.html">Trending</a>
-            <a href="market.html">Market</a>
-            <a href="about.html">About</a>
-            <div class="auth-links">
-                <a href="login.html">Login</a>
-                <a href="register.html">Register</a>
-            </div>
-        </div>
-    </nav>
-
-    <!-- Header -->
-    <header>
-        <h1>Welcome to the Investment App</h1>
-        <p>Your trusted platform for managing investments and staying up to date with market trends.</p>
-    </header>
+<body onload="loadUserDetails(); loadMarketOverview();">
+  <!-- Navigation Bar -->
+  <nav class="navbar">
+    <a href="#" class="logo">Investment App</a>
+    <div class="nav-links">
+      <a href="home.html">Home</a>
+      <a href="portfolio.html">My Portfolio</a>
+      <a href="notifications.html">Notifications</a>
+      <a href="trending.html">Trending</a>
+      <a href="market.html">Market</a>
+      <a href="about.html">About</a>
+      <div class="auth-links">
+        <a href="login.html">Login</a>
+        <a href="register.html">Register</a>
+      </div>
+    </div>
+  </nav>
 
-    <!-- Main Content -->
-    <main>
-        <!-- Latest Market News Section -->
-        <section class="section" id="market-news">
-            <h2>Latest Market News</h2>
-            <p>Stay updated with the latest market insights and trends.</p>
-        </section>
+  <!-- Header -->
+  <header>
+    <div id="welcome-message" class="welcome-container">
+      <h2>Welcome, <span id="username">User</span>!</h2>
+      <button onclick="toggleTheme()">Switch Theme</button>
+    </div>
+    <h1>Welcome to the Investment App</h1>
+    <p>Your trusted platform for managing investments and staying up to date with market trends.</p>
+  </header>
 
-        <!-- About Section -->
-        <section class="section" id="about">
-            <h2>About the Investment App</h2>
-            <p>Our app helps you manage your portfolio and provides real-time updates on market trends, news, and investment opportunities.</p>
-        </section>
-    </main>
+  <!-- Market Overview Section -->
+  <main>
+    <section class="market-container">
+        <h2>Market Overview</h2>
+        <div class="market-charts">
+          <!-- S&P 500 Chart -->
+          <div class="chart">
+            <h3>S&P 500</h3>
+            <canvas id="sp500Chart"></canvas>
+          </div>
+          <!-- NASDAQ Chart -->
+          <div class="chart">

```

Comment:
Maybe we condense these? Not sure what your were thinking would be different between the S&P and Nasdaq?

Reply:
Do you want me to have a radio button option to display both graphs or just have one? Overlay? Just wondering.